<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1231</width>
    <height>692</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>数据预测模块</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>170</y>
     <width>191</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>方法选择</string>
   </property>
   <widget class="QTableWidget" name="modelfuncs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>171</width>
      <height>221</height>
     </rect>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
   </widget>
  </widget>
  <widget class="QGroupBox" name="ingroup">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>241</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>模型介绍</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>1091</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>数据表</string>
   </property>
   <widget class="QTableView" name="init_data">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>1071</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>选择数据</string>
   </property>
   <widget class="QRadioButton" name="loaddata1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>101</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>计算数据</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="loaddata2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>导入数据</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>16</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="savep">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>420</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>保存参数</string>
   </property>
  </widget>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>490</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>logo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>442</x>
     <y>244</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>训练终点</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>442</x>
     <y>174</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>预测终点</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="predictdate">
   <property name="geometry">
    <rect>
     <x>522</x>
     <y>174</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2017</year>
     <month>10</month>
     <day>1</day>
    </datetime>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>442</x>
     <y>200</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>训练起点</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="startdate">
   <property name="geometry">
    <rect>
     <x>522</x>
     <y>200</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2015</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>442</x>
     <y>226</y>
     <width>252</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>（默认：以数据表中数据第一个值为训练起点）</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="enddate">
   <property name="geometry">
    <rect>
     <x>522</x>
     <y>244</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2017</year>
     <month>7</month>
     <day>1</day>
    </datetime>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>442</x>
     <y>270</y>
     <width>264</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>（默认：以数据表中数据最后一个值为训练终点）</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button2">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>600</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>全屏显示</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button3">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>629</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>保存数据</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button1">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>571</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确认分析</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="parameters">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>290</y>
     <width>271</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>模型参数</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="images">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>160</y>
     <width>501</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>图示</string>
   </property>
  </widget>
  <widget class="QTableView" name="result_table">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>440</y>
     <width>921</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="message">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>440</y>
     <width>181</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
